### ISSUE TITLE
Trouble opening ndtiff dataset

### ISSUE THREAD

**Issue by talonchandler** [czbiohub/iohub#105]
I'm running into issues opening some, but not all, recently acquired ndtiff datasets. 

Reproduce with:
```
python
>>> from iohub import read_micromanager
>>> reader = read_micromanager('/hpc/projects/compmicro/rawdata/mantis/2023_03_29_argolight/rings-LF-LS_1/rings-LF-LS_labelfree_1/')
Dataset opened
>>> reader.get_array(0)
```
fails with 
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/talon.chandler/iohub/iohub/ndtiff.py", line 182, in get_array
    return np.asarray(self.get_zarr(position))
  File "/home/talon.chandler/iohub/iohub/ndtiff.py", line 166, in get_zarr
    return da.reshape(shape)
  File "/home/talon.chandler/.conda/envs/recorder-test/lib/python3.9/site-packages/dask/array/core.py", line 2185, in reshape
    return reshape(self, shape, merge_chunks=merge_chunks, limit=limit)
  File "/home/talon.chandler/.conda/envs/recorder-test/lib/python3.9/site-packages/dask/array/reshape.py", line 248, in reshape
    inchunks, outchunks = reshape_rechunk(x.shape, shape, x.chunks)
  File "/home/talon.chandler/.conda/envs/recorder-test/lib/python3.9/site-packages/dask/array/reshape.py", line 62, in reshape_rechunk
    raise NotImplementedError(_not_implemented_message)
NotImplementedError:
Dask's reshape only supports operations that merge or split existing dimensions
evenly. For example:

>>> x = da.ones((6, 5, 4), chunks=(3, 2, 2))
>>> x.reshape((3, 2, 5, 4))  # supported, splits 6 into 3 & 2
>>> x.reshape((30, 4))       # supported, merges 6 & 5 into 30
>>> x.reshape((4, 5, 6))     # unsupported, existing dimensions split unevenly

To work around this you may call reshape in multiple passes, or (if your data
is small enough) call ``compute`` first and handle reshaping in ``numpy``
directly.
```

**Comment by ziw-liu** [comment]
Are the `ndtiff` (or `pycromanager`) versions in the acquisition environment and the analysis environment the same?

**Comment by ieivanov** [comment]
This is fixed in `ndtiff >= 1.12.1`, see https://github.com/micro-manager/NDTiffStorage/pull/107

**Comment by talonchandler** [comment]
Tested and it's working. Thanks @ziw-liu and @ieivanov!
